back/engines/commercial/analysis/app/serializers/analysis/web_api/v1/input_serializer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Analysis
+  module WebApi
+    module V1
+      class InputSerializer < ::WebApi::V1::BaseSerializer
+        set_type :analysis_input
+
+        attributes :title_multiloc, :body_multiloc, :custom_field_values, :published_at, :updated_at, :likes_count, :dislikes_count, :comments_count, :votes_count
+
+        belongs_to :author, serializer: ::WebApi::V1::UserSerializer
```

Comment:
This defined the relationships, correct? Are we missing the idea here when it's ideation?